### DISCUSSION TITLE
Clean up server edit screen

### DISCUSSION THREAD

**Discussion by camden-git** [pelican-dev/panel#47] [Ideas]
Currently the server edit screen is pretty cluttered and it's difficult to find what you're looking for.
![image](https://github.com/pelican-dev/panel/assets/64759244/57db46cd-20af-4270-964e-c66f04713fca)

It might be better to have something like the egg configuration page where everything is broken up into tabs.
![image](https://github.com/pelican-dev/panel/assets/64759244/97cb1f9e-e850-47cd-9336-912383737c40)

Pterodactyl itself in the older admin UI also does this, but it might be better to keep it as a single form similar to the egg UI as well.
![image](https://github.com/pelican-dev/panel/assets/64759244/82504145-7d0d-4875-957e-16df4679567f)

As plugins as on more and more having stuff already broken up into tabs will help down the line as upstream/plugins/forks add on new server configuration.

**Comment by lancepioch** [comment]
Thank you for your feedback, that page is not complete yet!
